src/store/actions.ts

```diff
@@ -318,7 +318,7 @@ export async function searchBusiness ({ getters }, corpNum: string): Promise<Bus
         if (data.directors === 'Not Available' && data.incorporated === 'Not Available') {
           return Promise.reject(error) // business not found
         }
-        // *** TODO: update this when new endpoint is available
+        // FUTURE: update this when new endpoint is available
```

Comment:
This isn't a "TODO" for us to work on. It's a "FUTURE" that Vysakh is working on.